.github/scripts/run_consensus.sh

```diff
@@ -2,21 +2,30 @@
 
 set -e
 
-mkdir -p docker/data/
-
-# source account ids
-source docker/env
-
-# generate chain spec and keystores based on authorites passed via --account-ids
-docker run -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e DAMIAN -e TOMASZ -e ZBYSZKO -e HANSU -e JULIA -e RUST_LOG=info \
- aleph-node:latest -c "aleph-node bootstrap-chain --base-path /data --account-ids $DAMIAN,$TOMASZ,$ZBYSZKO,$HANSU,$JULIA --sudo-account-id $DAMIAN > /data/chainspec.json"
+# change it when increasing number of node containers
+export NODE_COUNT=5
 
-# get bootnote peer id
-export BOOTNODE_PEER_ID=$(docker run -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e DAMIAN -e RUST_LOG=info aleph-node:latest -c "aleph-node key inspect-node-key --file /data/$DAMIAN/p2p_secret")
-
-echo "BOOTNODE_PEER_ID : $BOOTNODE_PEER_ID"
+mkdir -p docker/data/
 
-# Run consensus party
+echo "Generating accounts ids..."
+declare -a account_ids
+for node_index in $(seq 0 $((NODE_COUNT - 1))); do
+  account_ids+=($(docker run -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e node_index aleph-node:latest -c "aleph-node key inspect //$node_index | grep \"SS58 Address:\" | awk \"{print \\\$3;}\""))
```

Comment:
Are these `-e ...` for variables needed? Bash should substitute these values (works on my machine :))

Reply:
Done